framework/SupervisedLearning/ARMA.py

```diff
@@ -186,22 +186,30 @@ def __init__(self, messageHandler, **kwargs):
             continue
           self.fourierParams[v] = periods
       elif child.getName() == 'Peaks':
+        # read peaks information for each target
         peak={}
+        # creat an empty list for each target
         threshold = child.parameterValues['threshold']
         peak['threshold']=threshold
+        # read the threshold for the peaks and store it in the dict
         period = child.parameterValues['period']
         peak['period']=period
+        # read the period for the peaks and store it in the dict
         windows=[]
+        # creat an empty list to store the windows' information
```

Comment:
These comments work well; as a convention, though, we usually put the comments before the line. These are fine though, it's not a requirement.

Reply:
Okay, I will keep in mind next time